index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <h2>Hello world, we are software developers!</h2>
+    <div id="containerpage">
+      <header id="header">
+        <div class="hlogo">
+          <img
+            id="header-img"
+            src="images/triallogo.png"
+            ;
+            alt="original trombones logo"
+          />
+        </div>
+
+        <nav id="nav-bar">
+          <ul>
+            <li><a class="nav-link" href="#Features">Features</a></li>
+            <li><a class="nav-link" href="#How It Works">How It Works</a></li>
+            <li><a class="nav-link" href="">Pricing</a></li>
+          </ul>
+        </nav>
+      </header>
+      <div style="margin-top: 100px; margin-left: 50px">
```

Comment:
The professional way of coding inline style is not recommended.  would you add to the external CSS file.
```suggestion
      <div style="margin-top: 100px; margin-left: 50px">
```

Reply:
- [ ] Saying "The professional way" is too vague of a reason to request a change, please be specific on why using inline styles is not recommended and why the reviewed person could benefit from moving it into an external CSS file! 🤓 🔧